client/js/app/app.js

```diff
@@ -20,15 +23,17 @@ function App(config) {
   this.appDispatcher = AppDispatcher;
   this.targetNode = document.getElementById(config.targetId);
   this.persistence = null;
-  this.client = config.client;
+  this.client = new KeenAnalysis(config.client);
```

Comment:
So is the idea to no longer pass in an instantiated client object and instead hard binding to the version of `keen-analysis` we want intentional? I think this makes sense. I don't foresee many scenarios where a user would want to explicitly use a specific version of `keen.js` etc and this reduces risk of breakages with the wrong client version.